digiwf-integrations/digiwf-s3-integration/digiwf-s3-integration-client-example/src/main/java/io/muenchendigital/digiwf/s3/integration/example/client/controller/AllEndpointsUsageController.java

```diff
@@ -17,46 +16,45 @@
 @Slf4j
 @RestController
 @RequiredArgsConstructor
-@RequestMapping("/all")
 public class AllEndpointsUsageController {
 
-    private static final String LOG_DIVIDER = "------------";
-
-    private final ClientFileUsageController clientFileUsageController;
-
-    private final ClientFolderUsageController clientFolderUsageController;
-
-    @GetMapping
-    @ResponseStatus(HttpStatus.OK)
-    public void getFile() throws DocumentStorageException, DocumentStorageClientErrorException, DocumentStorageServerErrorException, IOException, PropertyNotSetException {
-        System.err.println("FILE usage");
-        this.clientFileUsageController.saveFile();
-        System.err.println(LOG_DIVIDER);
-        this.clientFileUsageController.updateFile();
-        System.err.println(LOG_DIVIDER);
-        this.clientFileUsageController.getFile();
-        System.err.println(LOG_DIVIDER);
-        this.clientFileUsageController.updateEndOfLife();
-        System.err.println(LOG_DIVIDER);
-        this.clientFileUsageController.deleteFile();
-
-        System.err.println("FILE INPUTSTREAM usage");
-        this.clientFileUsageController.saveFileInputStream();
-        System.err.println(LOG_DIVIDER);
-        this.clientFileUsageController.updateFileInputStream();
-        System.err.println(LOG_DIVIDER);
-        this.clientFileUsageController.getFileInputStream();
-        System.err.println(LOG_DIVIDER);
-        this.clientFileUsageController.updateEndOfLife();
-        System.err.println(LOG_DIVIDER);
-        this.clientFileUsageController.deleteFile();
-
-        System.err.println("FOLDER usage");
-        this.clientFileUsageController.saveFile();
-        System.err.println(LOG_DIVIDER);
-        this.clientFolderUsageController.getAllFilesInFolderRecursively();
-        System.err.println(LOG_DIVIDER);
-        this.clientFolderUsageController.deleteFolder();
-    }
-
+  private static final String LOG_DIVIDER = "------------";
+
+  private final ClientFileUsageController clientFileUsageController;
+
+  private final ClientFolderUsageController clientFolderUsageController;
+
+  @GetMapping("/all")
+  @ResponseStatus(HttpStatus.OK)
+  public void getFile() throws DocumentStorageException, DocumentStorageClientErrorException, DocumentStorageServerErrorException, IOException, PropertyNotSetException {
```

Comment:
don't know if we really need the example client.
An integration test instead would be better I guess?

Reply:
-> Future Story